GitUI/CommandsDialogs/BrowseDialog/FormUpdates.Designer.cs

```diff
@@ -30,24 +30,13 @@ private void InitializeComponent()
         {
             this.components = new System.ComponentModel.Container();
             System.ComponentModel.ComponentResourceManager resources = new System.ComponentModel.ComponentResourceManager(typeof(FormUpdates));
-            this.closeButton = new System.Windows.Forms.Button();
             this.UpdateLabel = new System.Windows.Forms.Label();
             this.progressBar1 = new System.Windows.Forms.ProgressBar();
             this.linkChangeLog = new System.Windows.Forms.LinkLabel();
-            this.btnDownloadNow = new System.Windows.Forms.Button();
+            this.btnUpdateNow = new System.Windows.Forms.Button();
+            this.linkDirectDownload = new System.Windows.Forms.LinkLabel();
             this.SuspendLayout();
             // 
-            // closeButton
-            // 
-            this.closeButton.Location = new System.Drawing.Point(456, 82);
-            this.closeButton.Margin = new System.Windows.Forms.Padding(4);
-            this.closeButton.Name = "closeButton";
-            this.closeButton.Size = new System.Drawing.Size(94, 29);
-            this.closeButton.TabIndex = 0;
-            this.closeButton.Text = "Close";
-            this.closeButton.UseVisualStyleBackColor = true;
-            this.closeButton.Click += new System.EventHandler(this.CloseButtonClick);
```

Comment:
The handler must be removed

Reply:
Hmm thought i removed those, good catch!